File: workoutpixel/src/test/java/com/karim/workoutpixel/CommonFunctionsTest.java
package com.karim.workoutpixel;

import com.karim.workoutpixel.core.CommonFunctions;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class CommonFunctionsTest {

    @Test
    public void next3AmTest() {

        assertEquals(CommonFunctions.timeBeautiful(CommonFunctions.next3Am()), "03:00");
        assertEquals(CommonFunctions.timeBeautiful(CommonFunctions.today3Am()), "03:00");
        assertEquals(CommonFunctions.timeBeautiful(CommonFunctions.last3Am()), "03:00");
    }
}

File: app/src/test/java/ch/karimattia/workoutpixel/CommonFunctionsTest.kt
package ch.karimattia.workoutpixel

import ch.karimattia.workoutpixel.core.timeBeautiful
import ch.karimattia.workoutpixel.core.next3Am
import ch.karimattia.workoutpixel.core.today3Am
import ch.karimattia.workoutpixel.core.last3Am
import org.junit.Assert
import org.junit.Test
import java.util.*

class CommonFunctionsTest {
	@Test
	fun next3AmTest() {
		val locale: Locale = Locale("ch", "de")
		Assert.assertEquals(timeBeautiful(next3Am(), locale = locale), "03:00")
		Assert.assertEquals(timeBeautiful(today3Am(), locale = locale), "03:00")
		Assert.assertEquals(timeBeautiful(last3Am(), locale = locale), "03:00")
	}
}